ComponentKit/Components/CKButtonComponent.h

```diff
@@ -33,7 +33,7 @@ struct CKButtonComponentAccessibilityConfiguration {
                     titleFont:(UIFont *)titleFont
                      selected:(BOOL)selected
                       enabled:(BOOL)enabled
-                       action:(CKComponentAction)action
+                       action:(const CKTypedComponentAction<UIEvent *> &)action
```

Comment:
Interesting. Passing by value allows correct capture in blocks. Perhaps we should just use that as the standard.